DigitalOffice/Tests/FileServiceTests/Repositories/AddNewFileRepositoryTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿using FileService.Database;
+using FileService.Database.Entities;
+using FileService.Repositories;
+using FileService.Repositories.Interfaces;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+using System;
+
+namespace FileServiceUnitTests.Repositories
+{
+    public class AddNewFileRepositoryTests
+    {
+        private DbFile newFile;
+        private IFileRepository repository;
+        private FileServiceDbContext dbContext;
+        private DbContextOptions<FileServiceDbContext> dbOptionsFileService;
+
+        [SetUp]
+        public void Initialization()
+        {
+            dbOptionsFileService = new DbContextOptionsBuilder<FileServiceDbContext>()
+                .UseInMemoryDatabase(databaseName: "FileServiceTestDatabase")
+                .Options;
+
+            dbContext = new FileServiceDbContext(dbOptionsFileService);
+            repository = new FileRepository(dbContext);
+
+            newFile = new DbFile
+            {
+                Id = Guid.NewGuid(),
+                Content = Convert.FromBase64String("RGlnaXRhbCBPZmA5Y2U="),
+                Extension = ".txt",
+                IsActive = true,
+                Name = "DigitalOfficeTestFile"
+            };
+        }
+
+        [Test]
+        public void SuccessfulAddNewFileToDatabaseTest()
+        {
+            Assert.AreEqual(newFile.Id, repository.AddNewFile(newFile));
+        }
```

Comment:
This test only lets you know that the return Id is correct, but you don't check if the file actually appeared in the database

Reply:
Well, i merged Develop and cannot find how to check otherwise. Suggestions? 
All tests at the moment just check the returned values.
@Inzilbeth

Reply:
Ill try to find the item in the database by its id, if result is not null then we're cool

Reply:
Huh! My bad! I indeed added such check, but forgot to add it in Successful Test. Fixed!